CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+# Important
+
+The Paradise GitHub is not exempt from Paradise Station community and server rules, especially rules 0, 1, and 4. An inability to abide by the rules on the GitHub will result in disciplinary action up to, or including, a repository ban.
+
+## General Expectations
+
+### PR Discussion
+
+Comments on Pull Requests should remain relevant to the PR in question and not derail discussions.
+
+Under no circumstances are users to be attacked for their ideas or contributions. While constructive criticism is encouraged, toxicity or general mean-spirited behaviour will not be tolerated. All participants on a given PR or issue are expected to be civil. Failure to do so will result in disciplinary action.
+
+"Merge-nagging" or similar behaviour is not acceptable. Comments of this nature will result in warnings or an outright ban from the repository depending on general behaviour.
+
+If you exhibit behaviour that's considered to be a net-negative to the community (offensive commentary, repeat violations, constant nagging, personal attacks, etc.), you may be banned from other Paradise services (discord, forums, server, wiki, etc.)
+
+Maintainers reserve the right to permanently revoke access from the repository if your behaviour is considered to be a net negative.
+
+### PR Approval/Objection Info
+
+Heads of Staff and Maintainers have the final say on Pull Requests. While thumbs ratios are generally taken into account, they do not dictate whether or not a PR will be merged.
+
+After a twenty four hour minimum waiting period, Pull Requests can be merged once they receive approval from both a Head of Staff and a Maintainer. An exception is made for refactors and fixes, which may be merged at a Maintainer's discretion with no waiting period.
+
+Heads of Staff and Maintainers are not obligated to publicly state their objections to a Pull Request. Attacking or berating either of these roles over an objection will not be tolerated. Additionally, whining over the closure of a PR, the existence of an objection, or similar behaviour, will not be tolerated.
```

Comment:
> Heads of Staff and Maintainers are not obligated to publicly state their objections to a Pull Request.

Since this is becoming officially official, I just want to state for the record that I disagree strongly with this part. (Under most circumstances anyway.)

Reply:
The reason we do this is because other maintainers got so much abuse from people nagging the moment we stated an objection. If people could behave, we wouldn't have to do this.